maven-plugin-api/src/main/java/org/apache/maven/plugin/Mojo.java

```diff
@@ -48,25 +46,4 @@
      */
     void execute()
         throws MojoExecutionException, MojoFailureException;
-
```

Comment:
not sure it makes much sense to drop them, the setLog is useful to keep a lazy init but more importantly, both are used by mojo so it will reduce migration cost (which is not handlable at maven level) to not break them IMHO

Reply:
back to the roots (as it as)